### ISSUE TITLE
ErrorException in style.cls.php line 506: Trying to get property of non-object

### ISSUE THREAD

**Issue by larsbo** [dompdf/dompdf#1008]
I got this exception:

``` php
ErrorException in style.cls.php line 506: Trying to get property of non-object in style.cls.php line 506
at HandleExceptions->handleError('8', 'Trying to get property of non-object', '/home/vagrant/myproject/vendor/dompdf/dompdf/include/style.cls.php', '506', array('length' => array('0.28571429rem'), 'ref_size' => '12', 'cache' => array(), 'key' => '0.28571429rem/12', 'ret' => '0', 'l' => '0.28571429rem', 'i' => '10')) in style.cls.php line 506
at Style->length_in_pt('0.28571429rem') in style.cls.php line 1989
at Style->_set_border_radius_corner('0.28571429rem', 'top_left') in style.cls.php line 1977
at Style->_set_border_radii('0.28571429rem', '0.28571429rem', '0.28571429rem', '0.28571429rem') in style.cls.php line 1969
at Style->set_border_radius('0.28571429rem') in style.cls.php line 704
at Style->__set('border-radius', '0.28571429rem') in stylesheet.cls.php line 1352
at Stylesheet->_parse_properties('background: #f3f4f5; padding: 0.71428571rem 1rem; box-shadow: none; border: 1px solid #d4d4d5; border-radius: 0.28571429rem;') in stylesheet.cls.php line 1380
at Stylesheet->_parse_sections('.ui.block.header { background: #f3f4f5; padding: 0.71428571rem 1rem; box-shadow: none; border: 1px solid #d4d4d5; border-radius: 0.28571429rem; }') in stylesheet.cls.php line 1127
...
```

The problem ist the following code:

``` php
$this->_stylesheet->get_dompdf()->get_tree()->get_root()->get_style()->font_size
```

**Comment by bsweeney** [comment]
Would help to also see the relevant HTML/CSS. Can you provide a test case the exhibits the problem?

**Comment by Tam** [comment]
I had a similar issue, and fixed it by removing all `rem` units.

**Comment by bsweeney** [comment]
I believe this is fixed in commit 1fa43aa08ab5264b1d3713afa9ed96f886bd4638.
